Hi @nicolaferraro. 
YAKS tests fail for me
```
test-crud-br5t4i6evfqbpljjt660 test [INFO] 
test-crud-br5t4i6evfqbpljjt660 test [INFO] Results:
test-crud-br5t4i6evfqbpljjt660 test [INFO] 
test-crud-br5t4i6evfqbpljjt660 test [ERROR] Errors: 
test-crud-br5t4i6evfqbpljjt660 test [ERROR]   Failed to check Http condition - request url 'http://api.yaks-a8ba4fb9-ca8f-46eb-955e-b60b392e5eb3.svc.cluster.local' did not return expected status '200'
test-crud-br5t4i6evfqbpljjt660 test [ERROR]   Failed to check Http condition - request url 'http://api.yaks-a8ba4fb9-ca8f-46eb-955e-b60b392e5eb3.svc.cluster.local' did not return expected status '200'
test-crud-br5t4i6evfqbpljjt660 test [ERROR]   Failed to check Http condition - request url 'http://api.yaks-a8ba4fb9-ca8f-46eb-955e-b60b392e5eb3.svc.cluster.local' did not return expected status '200'
test-crud-br5t4i6evfqbpljjt660 test [ERROR]   Failed to check Http condition - request url 'http://api.yaks-a8ba4fb9-ca8f-46eb-955e-b60b392e5eb3.svc.cluster.local' did not return expected status '200'
test-crud-br5t4i6evfqbpljjt660 test [ERROR]   Failed to check Http condition - request url 'http://api.yaks-a8ba4fb9-ca8f-46eb-955e-b60b392e5eb3.svc.cluster.local' did not return expected status '200'
```
1) the default method for health check is `HEAD` see https://github.com/citrusframework/citrus/blob/3c5d8052e581d895bb360d5c2fef8205af024972/core/citrus-base/src/main/java/com/consol/citrus/condition/HttpCondition.java#L46
2) You should probably increase the timeout `Given HTTP request timeout is 60000 ms`

--- a/test/crud.feature
+++ b/test/crud.feature
@@ -4,7 +4,8 @@ Feature: the API allows CRUD operations on a S3 bucket
   Background:
     Given integration api is running
     Given URL: http://api.${YAKS_NAMESPACE}.svc.cluster.local
-    And URL is healthy
+    Given HTTP request timeout is 60000 ms
+    And wait for GET on path /
 
 
   Scenario: LIST objects

